### PR TITLE
Closes #224 | Add/Update Dataloader KVIS Thai OCR

### LINKED ISSUE #224: Create dataset loader for KVIS Thai OCR
Dataloader name: `kvis_th_ocr/kvis_th_ocr.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?kvis_th_ocr

|     Dataset| kvis_th_ocr |
|-------------|---|
| Description | The KVIS Thai OCR Dataset contains scanned handwritten version of all 44 Thai characters obtained from 27 individuals. It consisted of 1079 images from 44 classes (letters). This dataset consists of all Thai consonants with different writing styles of various people from ages between 16 and 75. Vowels and intonation are not taken into consideration for the dataset collected. |
| Subsets | - |
| Languages | tha |
| Tasks | Optical Character Recognition |
| License | Creative Commons Attribution 4.0 (cc-by-4.0) |
| Homepage | http://dx.doi.org/10.17632/8nr3pbdk5c.1 |
| HF URL | - |
| Paper URL | https://ieeexplore.ieee.org/document/8584876 |

### PR DESCRIPTION
**Pull request by raileymontalan**
Closes #224

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py` (please use only lowercase and underscore for dataset folder naming, as mentioned in dataset issue) and its `__init__.py` within `kvis_th_ocr` folder.
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_LOCAL`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py` or `python -m tests.test_seacrowd seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py --subset_id {subset_name_without_source_or_seacrowd_suffix}`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+_HOMEPAGE = "https://data.mendeley.com/datasets/8nr3pbdk5c/1"
+
+_LANGUAGES = ["tha"]
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_LOCAL = False
+
+_URLS = "https://data.mendeley.com/public-files/datasets/8nr3pbdk5c/files/e5730ffa-527c-4e06-ac76-aab758f63ae0/file_downloaded"
```
Comment (faridlazuarda) [inline seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py:69]: I ran the standard test script 

```
python -m tests.test_seacrowd seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py
```

But the download link for the data doesn't seem to work. Can you resolve this? Thank you!

Comment (raileymontalan) [reply seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py:69]: Hi @faridlazuarda, could you tell me a bit more about the issue? On my end, the datasets downloads as expected.

Comment (faridlazuarda) [reply seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py:69]: Hello, this is what I got:

```
ConnectionError: Couldn't reach https://data.mendeley.com/public-files/datasets/8nr3pbdk5c/files/e5730ffa-527c-4e06-ac76-aab758f63ae0/file_downloaded (ConnectionError(ProtocolError('Connection aborted.', ConnectionResetError(54, 'Connection reset by peer'))))

----------------------------------------------------------------------
Ran 1 test in 1.907s

FAILED (errors=1)
```

Comment (faridlazuarda) [reply seacrowd/sea_datasets/kvis_th_ocr/kvis_th_ocr.py]: It works perfectly now. Thanks!

**Review by faridlazuarda — APPROVED** [review]
LGTM!

**Review by yongzx — APPROVED** [review]
Tested and it runs! LGTM.

**Pull request merged**